requirements/prod.txt

```diff
@@ -23,6 +23,7 @@ sqlalchemy-redshift==0.8.6
 pydruid[sqlalchemy]~=0.6.2
 PyAthena[SQLAlchemy,Pandas]==2.5.2
 sqlalchemy-databricks==0.2.0
+clickhouse-sqlalchemy==0.2.1
```

Comment:
A new version was published just yesterday :smile:

Let's update this to v0.2.2. - https://github.com/xzkostyan/clickhouse-sqlalchemy/releases/tag/0.2.2

Reply:
hahaha, okay we can update it to 0.2.2

Reply:
now u can check them -- thanks